webauthn4j-core/src/main/java/com/webauthn4j/data/attestation/statement/TPMTHA.java

```diff
@@ -57,7 +58,7 @@ public boolean equals(@Nullable Object o) {
         if (o == null || getClass() != o.getClass()) return false;
         TPMTHA tpmtha = (TPMTHA) o;
         return hashAlg == tpmtha.hashAlg &&
-                Arrays.equals(digest, tpmtha.digest);
+                MessageDigest.isEqual(digest, tpmtha.digest);
```

Comment:
This is a method to check value equality. Not used for authentication and there is no risk of timing attack.